src/main/java/ua/hillel/moneymanager/controller/ExpenditureController.java

```diff
@@ -0,0 +1,58 @@
+package ua.hillel.moneymanager.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.boot.autoconfigure.neo4j.Neo4jProperties;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.*;
+import ua.hillel.moneymanager.dto.CategoryDto;
+import ua.hillel.moneymanager.dto.ExpenditureDto;
+import ua.hillel.moneymanager.entity.Category;
+import ua.hillel.moneymanager.repository.CategoryRepository;
+import ua.hillel.moneymanager.repository.ExpenditureRepository;
+import ua.hillel.moneymanager.service.expenditure.ExpenditureService;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@RestController
+@RequestMapping("/categories")
+@RequiredArgsConstructor
+public class ExpenditureController {
+
+    private final ExpenditureService expenditureService;
+
+    @PostMapping("/{categoryName}/expenditures/create")
```

Comment:
1. `@PostMapping("/{categoryName}/expenditures/create")` -> `@PostMapping`
2. Can you include `categoryName` to `expenditureDto` and remove path variable?